linux

```diff
@@ -33,6 +33,7 @@ mcr.microsoft.com/dotnet/core/sdk dotnet
 node
 node:18 node-18
 node:20 node-20
+node:22 node-22
```

Comment:
What triggers the updates to the github actions & README? Is that handled by the automation or do these need to be rebuilt locally and proposed in the PR?